generators/typescript/utils/core-utilities/fetcher/src/fetcher/stream-wrappers/Node18UniversalStreamWrapper.ts

```diff
@@ -171,10 +171,15 @@ export class Node18UniversalStreamWrapper<ReadFormat extends Uint8Array | Uint16
     public async text(): Promise<string> {
         const chunks: ReadFormat[] = [];
 
-        while (true) {
+        const infiniteRead = true;
```

Comment:
what is this?

Reply:
to get around the husky linting, can revert